src/components/Editor/SearchBar.js

```diff
@@ -304,6 +304,11 @@ class SearchBar extends Component {
       e.stopPropagation();
     }
 
+    // Check if searchType is "text"
+    if (!["functions", "variables"].includes(searchType)) {
+      this.props.toggleSymbolSearch(false);
```

Comment:
hmm, this seems a bit out of date given what we're doing now. Maybe we want it to be: `setEditorSearchType` and that can include text